src/common/servlist.c

```diff
@@ -138,7 +139,7 @@ static const struct defaultserver def[] =
 	{0,			"irc.inet.tele.dk"},
 
 	{"ElectroCode", 0, 0, 0, LOGIN_SASL, 0, TRUE},
-	{0,			"irc.electrocode.net"},
+	{0,			"irc.electrocode.net/+6697"},
```

Comment:
This already used ssl, thats what the last TRUE is above. Same should apply to buddyim.